src/openrct2/scenario/ScenarioRepository.cpp

```diff
@@ -150,9 +151,11 @@ class ScenarioRepository final : public IScenarioRepository
         std::string rct1dir = _env->GetDirectoryPath(DIRBASE::RCT1, DIRID::SCENARIO);
         std::string rct2dir = _env->GetDirectoryPath(DIRBASE::RCT2, DIRID::SCENARIO);
         std::string openrct2dir = _env->GetDirectoryPath(DIRBASE::USER, DIRID::SCENARIO);
+        std::string rct1datdir = _env->GetDirectoryPath(DIRBASE::RCT1, DIRID::DATA);
         Scan(rct1dir);
         Scan(rct2dir);
         Scan(openrct2dir);
+        ScanMegaPark(rct1datdir, rct1dir);
```

Comment:
Unsure as to whether we should put the decoded SC4 in the RCT1 dir, or keep it in the user's OpenRCT2 scenarios folder. We don't usually modify the source directories, but then again we're not doing anything that the original game didn't do here.